docs/installation.rst

```diff
@@ -133,20 +133,19 @@ The following Python packages are required:
 
 * cryptography>=3.3.2
 * tornado>=5.0.2
-* m2crypto>=0.21.1
 * pyzmq>=14.4
 * pyyaml>=3.11
-* simplejson>=3.8
 * requests>=2.6
 * sqlalchemy>=1.3
 * alembic>=1.1.0
-* python-gnupg>=0.4.6
+* packaging>=20.0
```

Comment:
Remove the one below if `packaging>=16.0`?

Reply:
Yes, I missed that.